vendor/bat-native-ledger/src/bat/ledger/internal/database/database_contribution_queue.cc

```diff
@@ -87,20 +89,18 @@ void DatabaseContributionQueue::OnInsertOrUpdate(
     return;
   }
 
-  auto queue =
-      braveledger_bind_util::FromStringToContributionQueue(queue_string);
-
-  if (!queue) {
+  if (!shared_queue) {
     BLOG(0, "Queue is null");
     callback(ledger::Result::LEDGER_ERROR);
     return;
   }
 
+  auto queue = std::move(*shared_queue);
```

Comment:
Is this necessary?

Reply:
sadly without this it's not working, do you maybe have a way how to remove this? compiler was complaining a lot :)

Reply:
found a way 😄 fixing now

Reply:
It seemed like you could just do this, but maybe I'm wrong:

```
  publishers_->InsertOrUpdate(
      shared_queue->id,
      std::move(shared_queue->publishers),
      callback);
```

Reply:
Awesome!